packages/styles/src/components/atoms/_loader.scss

```diff
@@ -34,4 +35,115 @@
   &--default {
     color: $--tk-loader-default !important;
   }
+
+  &--small {
+    font-size: toRem(12);
+  }
+
+  &--medium {
+    font-size: toRem(16);
+  }
+
+  &--large {
+    font-size: toRem(32);
+  }
+}
+
+.tk-loader-textPos--right {
+  p {
+    margin: 0 toRem(8);
+  }
+}
+.tk-loader-textPos--bottom {
+  p {
+    margin: toRem(8) 0;
+  }
+}
+.tk-loader-container {
+  display: flex;
+}
+.tk-loader-textPos {
+  &--bottom {
+    display: inline-block;
+    text-align: center;
+  }
+  &--right {
+    display: flex;
+    align-items: center;
+  }
+}
+.tk-loader-text {
+  width: toRem(59);
+  color: $--tk-main-text-color;
+  font-family: 'Segoe UI';
+  font-style: normal;
+  font-weight: 400;
+  font-size: toRem(14);
+  line-height: toRem(20);
+  margin: 0;
+}
+
+// Progressbar
+
+.tk-loader {
```

Comment:
Could you try to find a way to factorize this? 
read about https://sass-lang.com/documentation/at-rules/mixin 
or  https://sass-lang.com/documentation/at-rules/function